### DISCUSSION TITLE
v0.22.1

### DISCUSSION THREAD

**Discussion by alex-w** [Stellarium/stellarium#2402] [General]
**The installers are located in the "Assets" section at the end of these release notes!**

[![Github Release](https://img.shields.io/github/downloads/Stellarium/stellarium/v0.22.1/total.svg)](https://github.com/Stellarium/stellarium/releases/v0.22.1)

A huge thanks to the people who helped us a lot by reporting bugs!

## Release notes
The Stellarium team is delighted to announce the release of Stellarium 0.22.1.

The major changes of this version:
- Added fixed equatorial grid/line
- Added new sky cultures
- Fixed building Stellarium (AppImage)
- Updated DSO catalog

Enjoy!

## What's Changed
Full list of changes:
- Added Khoikhoi and San sky culture (GH: #2396)
- Added Xhosa sky culture (GH: #2395)
- Added fixed equatorial grid/line (GH: #2399)
- Added DSO names (GH: #2387)
- Fixed building AppImage/x86_64 package
- Fixes various typos in sky cultures (GH: #2400)
- Fixed rendering 3D models for solar system bodies (GH: #2367)
- Changed screenshots naming: properly detect largest screenshot number (GH: #2379)
- Updated DSO catalog (v3.15):  updated visual magnitudes for Messier objects from RNGC/IC catalog by Wolfgang Steinicke (GH: #2375)
- Updated designations of stars (GH: #2391)
- Updated licence statement for few sky cultures

## Packages notes
package | note
------------|---------
stellarium-0.22.1.tar.gz | Stellarium 0.22.1 for UNIX/Linux (source code). <br />__MD5:__ c0f9822386d09485ea2e3e031d0cbcdb <br />__SHA1:__ 55b5915dfb616b9748e1c2d2de6150f7159cf36c <br />__SHA256:__ 61a595ecc345dcc7517359573d0dc772e67ecaf6d52fefab142a77c92cf3fa24
stellarium-0.22.1-win32.exe | Stellarium 0.22.1 for Windows (32-bit; Windows 7+) based on Qt 5.12.6 (Visual Studio 2017 Community) with media support. <br /> __MD5__: 4ee79d430dbcd9250541bca207d9ae53 <br />__SHA1__: 6b682478e3216c453b63de31fa5425d67778ebde <br />__SHA256__: 720f340800370dbf03547211ce567c48f0f2aa552cf3e5dc98a095053a49377a
stellarium-0.22.1-win64.exe | Stellarium 0.22.1 for Windows (64-bit; Windows 7+) based on Qt 5.12.6 (Visual Studio 2017 Community) with media support. <br /> __MD5__: f9c4456d4c2fd690645b354588d71582 <br />__SHA1__: ebe05e3d0682e24aa270c2d9fcb67bd177b33942 <br />__SHA256__: 769e647a6b977ae6b03bc6b9290552bc5ad6a9071b53a7fee7f5291b9229e396
Stellarium-0.22.1-x86_64.zip | Stellarium 0.22.1 for Mac OS X (x86_64; Mac OS X 10.12+) based on Qt 5.12.9 (AppleClang 11.0.0) with media support. <br /> __MD5__: 8b4c25bfc98d841a07d5dcd73a94b7ff <br />__SHA1__: e7e98ab0d6d1aac72f069a6ed871562d555368e5 <br />__SHA256__: a3993feda9b9401f5838ff1c43f1d3c22d9c9ef2c085015f4e6b4a9447b73924
Stellarium-0.22.1-arm64.zip | Stellarium 0.22.1 for Mac OS X (arm64; Mac OS X 10.12+) based on Qt 5.15.3 (AppleClang 13.1.6) with media support. <br /> __MD5__: 75eef016fd037eeee2d78e2807053418 <br />__SHA1__: ea4810a81552aa8d5a84d3b4d538e5a8172cc7ea <br />__SHA256__: a9a1bb92b6ec8df2722da57c1d3e5162f9ca0d4b12fbc642c7ee0d947ffa9b53
Stellarium-0.22.1-x86_64.AppImage | Stellarium 0.22.1 for Linux (AMD64; glibc 2.27) based on Qt 5.9.5 (GCC 7.5.0) with media support. <br /> __MD5__: 4b4d85d0913714eaa70cc1ac7a516dfd <br />__SHA1__: 09249621f5c480aea4927103147dc3c845377f38 <br />__SHA256__: 7ae0ac7a9162f8800ae284c7b4e27ba2023cbe5a6ba610774abec84515670433

<hr /><em>This discussion was created from the release <a href='https://github.com/Stellarium/stellarium/releases/tag/v0.22.1'>v0.22.1</a>.</em>

**Comment by ultrapre** [comment]
Congratulations! But why 0.22.1 published so fast?

**Reply by alex-w** [reply]
A hotfix with resolving an infrastructure issues...

**Comment by hschneck** [comment]
I'm having troubles with running Stellarium 0.22.1 on my MacBook Pro with Monterey 12.3.1.  It is ridiculously slow to respond to trackpad clicks.  Before I upgraded to Monterey I did not have this problem (and was on an earlier version of Stellarium). I have an i9 with 64 GB RAM and a 1 TB SDD.   Has anyone else run into this and have any idea what might be causing this.  I have not other apps running.

**Reply by alex-w** [reply]
I can’t reproduce the issue, but I have macOS 12.4 on MBP M1

**Comment by astro-moose** [comment]
This problem persists in Stellarium 1.2 /25.12.2022) on MacOS 12.6.2 with a MacBook Pro 2015. Using the trackpad is a pain in the ass, stellarium nearly freezes. The same Application runs flawless on a hacked Mac Mini 2012 with 12.6.2.

Using a mouse on the Macbook helps a little bit but this cannot be the solution...

**Reply by 10110111** [reply]
Sounds like #2778.
